pkg/boot/grub/grub.go

```diff
@@ -126,6 +133,31 @@ func grubScanBLSEntries(mountPool *mount.Pool, variables map[string]string) ([]b
 	return images, nil
 }
 
+// Find and return the value of the key from a grubenv file
+func findkeywordGrubEnv(file string, fsRoot string, key string) string {
+	FileGrubenv, err := filepath.Glob(filepath.Join(fsRoot, file))
+	if FileGrubenv == nil || err != nil {
+		return ""
+	}
+	relNamesFile, err := os.Open(FileGrubenv[0])
+	if err != nil {
+		log.Printf("[grubenv] Could not Open %s: %v", FileGrubenv[0], err)
```

Comment:
in general, packages should not print. Please rewrite this function to return (string, error)

Reply:
done